modules/admin_manual/pages/release_notes.adoc

```diff
@@ -61,7 +61,7 @@ For code cleanup reasons, the execution of background jobs (e.g., for public lin
 
 * If you're using https://doc.owncloud.com/server/admin_manual/configuration/server/background_jobs_configuration.html#cron[System cron] to trigger background job execution, there is a new `occ` command (`occ system:cron`) which executes the background jobs. 
   To make use of it, you have to change the entry in `crontab`. 
-  Instead of executing `cron.php` (e.g., `/usr/bin/php -f /path/to/your/owncloud/cron.php`), cron should use `occ system:cron` (e.g., `/usr/bin/php -f /path/to/your/owncloud/occ system:cron`). 
+  Instead of executing `cron.php` (e.g., `/usr/bin/php -f /path/to/your/owncloud/cron.php`), cron should use `occ system:cron` (e.g., `{occ-command-example-prefix} system:cron`). 
```

Comment:
Shouldn´t the {occ...} also be set in the example?

Reply:
Sory, but I don't follow what you're asking.

Reply:
Sorry, you are right. Have overseen that cron.php was not called by occ... 🤦‍♂